lib/api/providers/client.ts

```
@@ -0,0 +1,41 @@
+'use client';
+
+import { useApiClient } from '@/lib/api-client/client';
+import { ApiProvider, PROVIDER_API, transformProvider } from '../providers';
```

ESLint

[error] 'ApiProvider' is defined but never used.

(@typescript-eslint/no-unused-vars)